src/forms/StaticField.tsx

```diff
@@ -1,25 +1,27 @@
 import { useId } from "@react-aria/utils";
 import { ReactNode } from "react";
-import { Css } from "src/Css";
+import { PresentationFieldProps } from "src/components/PresentationContext";
+import { Css, px } from "src/Css";
 import { defaultTestId } from "src/utils/defaultTestId";
 import { useTestIds } from "src/utils/useTestIds";
 
 interface StaticFieldProps {
   label: ReactNode;
   value?: string;
   children?: ReactNode;
+  labelStyle?: PresentationFieldProps["labelStyle"];
 }
 
 export function StaticField(props: StaticFieldProps) {
-  const { label, value, children } = props;
+  const { label, labelStyle = "above", value, children } = props;
```

Comment:
Suggestion: Can you initialize this from the presentation context?

```
  const { fieldProps } = usePresentationContext();
  const { label, 
    labelStyle = fieldProps?.labelStyle ?? "above",
```

Like we do in `TextFieldBase`? That way it will get the right default is someone is using this inside of a FormLines or other PresentationContext

Reply:
Nice, good catch, thank you! 👏